### ISSUE TITLE
Issue when including Orion key in Vendor hybrid package

### ISSUE THREAD

**Issue by brt5586** [JakedUp/Syncler-Packages#12]
Hi man, seems to be a recent issue where if you provide the API key for Orion in the GitHub page and I checked the vendor JSON details and the manifest JSON property which takes you directly to the package works when setting exclude Orion and shows the JSON for said package.

however after setting it up for Orion key this is the manifest URL it gives in the vendor JSON:
https:\/\/syncler-providers.herokuapp.com\/syncler\/express-hybrid.json?mode=manifest&orion-api-key=REDACTED

tried this URL directly and it’s a blank page but when excluded it’s this for URL:

https:\/\/syncler-providers.herokuapp.com\/syncler\/express-hybrid.json?mode=manifest&exclude-orion=true

This returns a full package JSON but the other doesn’t meaning Syncler doesn’t actually install any packages and installs only the Vendor and then going into available packages returns 0 results

**Comment by brt5586** [comment]
To add it appears when the API key is too long the page is blank. I recently refreshed my Orion API key so might be too long for it to understand?

**Comment by brt5586** [comment]
Appears this may be working now @JakedUp  did you manage to fix this now?

**Comment by brentonv** [comment]
It was never broken

**Comment by JakedUp** [comment]
I don't think this was ever an issue. You might have been using an incorrect ORION key.
